docs/reference/analysis/charfilters/mapping-charfilter.asciidoc

```diff
@@ -4,6 +4,16 @@
 A char filter of type `mapping` replacing characters of an analyzed text
 with given mapping.
 
+[horizontal]
+`mappings`::
+
+    A list of mappings to use.
+
+`mappings_path`::
+
+    A path (either relative to `config` location, or absolute) to a mappings
+    file configuration.
```

Comment:
I think the best way is to make absolute path deprecated because it must be inside `CONF_DIR`. But IMO this must be new PR (deprecation warning and fix for all docs for token and character filters with `*_path` settings). 

The phrase "either relative to `config` location, or absolute" is same for almost every `*_path` setting in token filters docs, so, for now, it should be the same as others to make future editing more simple.

Nevertheless, this note should be on every `*_path`, not only in `mapping_path` filter description.

@nik9000 what do you think about this ?